src/quick-lint-js/diag/diagnostic-types.h

```diff
@@ -2976,13 +2986,6 @@
       diag_unexpected_colon_after_generic_definition, "E0331",                  \
       diagnostic_severity::error, { source_code_span colon; },                  \
       MESSAGE(QLJS_TRANSLATABLE("':' should be 'extends' instead"), colon))     \
-                                                                                \
-  QLJS_DIAG_TYPE(                                                               \
```

Comment:
Must fix: I think you reverted someone else's commit by mistake (edec0892d64aad3dc76f5d1e4fa4f4931ecf9993?). Reintroduce this diagnostic.